### ISSUE TITLE
Failed to load asset manifest

### ISSUE THREAD

**Issue by rwwagner90** [ember-engines/ember-engines#486]
This seems to be an issue with fastboot and engines. I get:

`Error: Failed to load asset manifest. For browser environments, verify the meta tag with name "website/config/asset-manifest" is present. For non-browser environments, verify that you included the node-asset-manifest module.`

I have `lazyLoading: false`, but I'm not sure if there is anything I am missing with configuring for fastboot? Unsure what the issue is here.

**Comment by rwwagner90** [comment]
FYI The problem seems to go away if I downgrade `ember-engines` to `0.5.9`

**Comment by trentmwillis** [comment]
As far as I know, Engines don't work with FastBoot. Since Engines read information from the DOM (for `config/asset-manifest` and `config/environment`), they will error out by default unless FastBoot is configured to load them.

This PR should address most issues: https://github.com/ember-engines/ember-engines/pull/444. Once that is ready to go, we'll need to squash individual bugs one at a time with reproductions as there will likely be some combinations of ember-cli + ember-engines + ember.js + fastboot that are missed (dependencies are fun!).

**Comment by rwwagner90** [comment]
@trentmwillis it seems that it does work, at least with `ember s` when I downgrade to `ember-engines` `0.5.9`, but then when I deploy, I still get the error. Is there a way to manually configure fastboot to work with engines now or do I need to wait on those PRs?

**Comment by trentmwillis** [comment]
You can use the `updateFastBootManifest` hooks provided by FastBoot, but you'll pretty much be on your own. I'm hoping to land the necessary changes here soon (within the next week), just waiting on another blocking PR to get merged first.

**Comment by rwwagner90** [comment]
I can wait a week or so, no worries 😃

**Comment by valtermartins1301** [comment]
Hy guys, i'm not using fastboot on my project and i'm getting the same Error. 
I've already tried to run a fresh project but the same issue occurs... :disappointed: 

What i need to do to workaround this problem ?

**Comment by rwwagner90** [comment]
@valtermartins1301 I think fastboot just is not compatible with engines until @trentmwillis finishes his awesome work making the two play nicely together 😄

**Comment by marcemira** [comment]
Same as @valtermartins1301, I don't have anything fastboot related and I'm getting this error, any clues on what could be wrong? using current 0.5.14, thanks!

**Comment by jherdman** [comment]
Perhaps this is related to the autoboot flag in 2.18?

**Comment by marcemira** [comment]
I wasn't able to fix it, but got rid of the issue for now by starting a project from scratch, using the `ember new foo-engine -b https://github.com/ember-engines/engine-blueprint.git` method, still not sure what was the cause :/

**Comment by vinngn1** [comment]
I tried create a new project, the problem was gone. I noticed that inside the `package.json dependencies`, there is no `ember-engines`. I tried to add `ember-engines` as dependency, the manifest issue occurs again.

Without ember-engines as dependency, it fails to recognize a component from another addon which include as an dependency for the engine.

Uncaught Error: Could not find module `ember-engine-my-profile/components/user-profile` imported from `(require)`

Now, I am totally stranded :(
